src/thor/matrix_action.cc

```diff
@@ -6,6 +6,7 @@
 #include "thor/timedistancematrix.h"
 #include "thor/worker.h"
 #include "tyr/serializers.h"
+#include <valhalla/worker.h>
```

Comment:
this makes me want to pull the warning stuff out of the worker.h. tons of unneeded stuff being compiled in here..